app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/YouCannotUseServiceController.scala

```diff
@@ -37,7 +37,7 @@ class YouCannotUseServiceController @Inject() (
   override val env: Environment,
   override val authConnector: AuthConnector,
   authAction: AuthAction,
-  cache: SessionCache,
+  subscriptionBusinessService: SubscriptionBusinessService,
```

Comment:
I think `SubscriptionBusinessService` uses SessionCache itself, but I think it makes sense to use Service layer here instead of SessionCache directly